scio-core/src/main/scala/com/spotify/scio/accumulators/package.scala

```diff
@@ -79,8 +79,8 @@ package object accumulators {
 
     /** Count positive and negative results in a filter with automatically created accumulators. */
     def accumulateCountFilter(f: T => Boolean): SCollection[T] = {
-      val accPos = self.context.sumAccumulator[Long]("Positive#" + CallSites.getCurrentName)
-      val accNeg = self.context.sumAccumulator[Long]("Negative#" + CallSites.getCurrentName)
+      val accPos = self.context.sumAccumulator[Long]("Positive#" + self.tfName)
```

Comment:
These 2 are not `PTransform` names but accumulator identifiers. We should keep the original so that they're distinguishable.